docker-compose.yml

```diff
@@ -253,7 +250,7 @@ services:
       S3_STATUS_FOLDER: "status"
       S3_SERVICE_ENDPOINT: "http://s3-dummy:4572"
       NIFI_ROOT_FOLDER: "/data/output" # see the shared-volume above
-      NIFI_FILE_NAMES_CSV: "db.core.addressDeclaration-000001.txt.bz2,db.quartz.claimantEvent-000001.txt.bz2"
+      NIFI_FILE_NAMES_CSV: "db.core.addressDeclaration-045-050-000001.txt.gz,db.quartz.claimantEvent-045-050-000001.txt.gz"
```

Comment:
where does `045-050` come from?

Reply:
It's the range of bytes that the first character of the ids in this batch are in.